src/components/common/v2/TextBox/PopUp.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+import { useState } from 'react';
+
+import Icon from '../../Icon';
+
+import { theme } from '@/styles/theme';
+
+const TYPE = {
+	TITLE: 'title',
+	DESC: 'description',
+} as const;
+
+const STATE = {
+	PLACEHOLDER: 'placeholder',
+	DEFAULT: 'default',
+	TYPING: 'typing',
+	FIELD: 'field',
+} as const;
+
+type PopUpProps = {
+	type: (typeof TYPE)[keyof typeof TYPE];
+};
+
+function PopUp({ type }: PopUpProps) {
+	const [state, setState] = useState<(typeof STATE)[keyof typeof STATE]>(STATE.DEFAULT);
+
+	const handleFocus = (e: React.FocusEvent<HTMLInputElement>) => {
+		if (!e.target.value) {
+			setState(STATE.PLACEHOLDER);
+		}
+	};
+
+	const handleBlur = (e: React.FocusEvent<HTMLInputElement>) => {
+		if (e.target.value) {
+			setState(STATE.FIELD);
+		} else {
+			setState(STATE.DEFAULT);
+		}
+	};
+
+	const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+		if (e.target.value) {
+			setState(STATE.TYPING);
+		}
+	};
+
+	return (
+		<PopUpContainer>
+			{type === TYPE.DESC && state === STATE.DEFAULT && <Icon name="IcnDescription" size="tiny" />}
+			<StyledInput
+				type={type}
+				state={state}
+				placeholder="제목을 입력하세요"
+				onFocus={handleFocus}
+				onBlur={handleBlur}
+				onChange={handleChange}
+			/>
+		</PopUpContainer>
+	);
+}
+
+export default PopUp;
+
+const PopUpContainer = styled.div`
+	display: flex;
+	flex-direction: row;
+	gap: 8px;
+	align-items: center;
+`;
+
+const StyledInput = styled.input<{ type: PopUpProps['type']; state: (typeof STATE)[keyof typeof STATE] }>`
+	${({ type }) => (type === TYPE.TITLE ? theme.font.title02 : theme.font.body03)};
```

Comment:
```suggestion
	${({ type, theme }) => (type === TYPE.TITLE ? theme.font.title02 : theme.font.body03)};
```

theme 를 직접 임포트하는 것보다 일케 같이 써주면 컨텍스트에서 제공되는 스타일을 써줄 수 있을 것 같아요~!!

Reply:
반영했습니다!